### DISCUSSION TITLE
Only see Stat  in TreeMap sizes && Error parsing bundle asset "#*.js": no such file

### DISCUSSION THREAD

**Discussion by kanishk30** [webpack-contrib/webpack-bundle-analyzer#579] [Q&A]
#### Issue description
I can only see Stat after running bundle analyser. 

<img width="755" alt="Screenshot 2021-11-23 at 7 16 44 PM" src="https://user-images.githubusercontent.com/34736432/143035682-ae27be35-4793-48f0-b739-148c0a4f6cf4.png">

Just adding following line makes it happen,
**'react-dates/lib': 'react-dates/esm'**
```
 resolve: {
      modules: [path.resolve(__dirname), 'node_modules'],
      alias: {
        ....
        'react-dates/lib': 'react-dates/esm'
      },
```

If I remove this line, I am able to see all three ( Stat, Parse, Gzipped) 

Logs 
```

Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/listings.ca000d74c8c3d444f5bb.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/383.c4f37401fce0d5d17251.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/513.3ee8770c259eb4dc40d3.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/fa.6912da9faf08c2443762.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/9532.08fdeb4bb3242bf5dd03.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/476.aa246cd3d41a38e57f5a.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/6350.cc4d2249c16bac0b5c7f.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/289.8b1937750209e918ca5e.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/promotions.c1ddfa1417ee87b1e8e7.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/238.64dc60aa00b757923af9.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/5639.93c6c9816b101f4d2f57.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/397.fa964c3bde40237bcda9.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/953.d0fb03309efcbc95434b.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/fbflite.4b58541d0a3fa775ba92.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/850.5cab811dc87b85c165b9.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/546.ddc0bda42bcd377f330d.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/app.c5465342fad49a75af26.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/9072.e6be74f73f8427a09579.js": no such file
Error parsing bundle asset "/Users/kanishk.agrawal/workspace/sp-seller-dashboard/client/dist/bundles/partnerServices.9e9d449061b978da4aeb.js": no such fil

```

#### Technical info
`package.json`
```

  "scripts": {
  ....,
    "webpack2650MB": "node --max_old_space_size=2560 node_modules/.bin/webpack",
    "webpack4096MB": "node --max_old_space_size=4096 node_modules/.bin/webpack",
    "prod": "npm run webpack2650MB -- --env.prodEnvironment",
    "prod:profile": "npm run prod -- --env.profile",
    ....
  },
```
```
  "devDependencies":  {
  ....,
      "webpack": "5.43.0",
    "webpack-bundle-analyzer": "3.6.1",
    "webpack-livereload-plugin": "0.11.0",
    "webpack-merge": "4.1.0",
    "webpack-notifier": "1.13.0",
    "webpack-sources": "1.0.1",
    "webpack-stats-plugin": "0.3.1",
    "webpack-visualizer-plugin": "0.1.11"
    ....
}
```
Major dependency - 
```
react: 17.0.1
react-dates: 21.6.0
"webpack-cli": "^3.3.6",
```

<pre>
  System:
    OS: macOS 11.6
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Memory: 1.21 GB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.17.3 - /usr/local/bin/node
    Yarn: 1.22.0 - /usr/local/bin/yarn
    npm: 6.14.13 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  npmPackages:
    clean-webpack-plugin: 0.1.16 => 0.1.16 
    copy-webpack-plugin: ^4.5.1 => 4.6.0 
    terser-webpack-plugin: ^4.2.3 => 4.2.3 
    uglifyjs-webpack-plugin: 2.2.0 => 2.2.0 
    webpack: 5.43.0 => 5.43.0 
    webpack-bundle-analyzer: 3.6.1 => 3.6.1 
    webpack-cli: ^3.3.6 => 3.3.10 
    webpack-livereload-plugin: 0.11.0 => 0.11.0 
    webpack-merge: 4.1.0 => 4.1.0 
    webpack-notifier: 1.13.0 => 1.13.0 
    webpack-sources: 1.0.1 => 1.0.1 
    webpack-stats-plugin: 0.3.1 => 0.3.1 
    webpack-visualizer-plugin: 0.1.11 => 0.1.11 
</pre>

#### Debug info
It is a huge app. I am trying to shrink react-dates bundle size. In the process , i added **'react-dates/lib': 'react-dates/esm'** following this comment ( https://github.com/airbnb/react-dates/issues/874#issuecomment-360878196 ) ; but after this I cannot see Gzipped in chart generated by bundle analyser.

**Comment by alexander-akait** [comment]
Duplicate https://github.com/webpack-contrib/webpack-bundle-analyzer/issues/476

**Comment by valscion** [comment]
Doesn't seem like a duplicate of #476 to me. However, the information provided is not enough to be able to reproduce this bug and fix it 😞

**Comment by kanishk30** [comment]
@valscion - thanks for responding. Pls tell me what more info i can provide. i tried some debugging. i noticed that adding an alias for `lib > esm` made this issue happen. so, could it be because webpack-bundle-analyser doesnt identify alias? or soething to with es module?

**Comment by valscion** [comment]
I have no idea. Without a minimal sized repository that shows the issue, debugging further is nearly impossible to me
